Identity/Identity.API/Helpers/JWTHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+using System.IdentityModel.Tokens.Jwt;
+
+using Microsoft.IdentityModel.Tokens;
+using Identity.DataAccess.Models.Entities;
+
+namespace Identity.API.Helpers;
+
+public class JWTHelper
+{
+    public static string GenerateJwtToken(UserEntity user)
+    {
+        DateTime value = DateTime.Now.AddMinutes(20.0);
+        byte[] bytes = Encoding.UTF8.GetBytes(ApplicationSettingsHelper.Settings.GetSection("JWTSettings:Key").Value!);
+        SigningCredentials signingCredentials = new SigningCredentials(new SymmetricSecurityKey(bytes), SecurityAlgorithms.HmacSha256);
+
+        Dictionary<string, object> claims = new Dictionary<string, object> { { "id", user.Id.ToString() }, { "roles", ((int)user.Roles).ToString() } };
```

Comment:
I would prefer `roles` to be a list of strings, instead of the binary flags. This will make more sense where the tokens will be consumed.

Reply:
Also would like to have the `username` here as a claim as well. We'd be able to make us of showing that value in the front end

Reply:
Don't know if I fully agree with the roles change but implemented it anyway.

Reply:
In the entity you could've kept the flag enum :D i was talking about returning the list of strings in the claims only